server/prepare.c

```diff
@@ -338,12 +341,12 @@ bool send_data_header(
     const struct captured_fields *fields,
     const struct data_capture *capture,
     const struct data_options *options,
-    struct buffered_file *file, uint64_t missed_samples)
+    struct buffered_file *file, uint64_t missed_samples, char *timestamp_message)
 {
     struct xml_element header =
         start_element(file, "header", options->xml_header, true, true);
 
-    send_capture_info(file, capture, options, missed_samples);
+    send_capture_info(file, capture, options, missed_samples, timestamp_message);
```

Comment:
Do the `struct ts` to string formatting here, please.

Reply:
Actually, I don't think I mean here, I mean where you were doing it the first time, in `send_capture_info`.